src/components/images/EditImage.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import LocalApi from "../../apis/LocalApi";
+import { Form, Button, Container } from "react-bootstrap";
+
+class EditImage extends Component {
+  state = {
+    selectedFile: null,
+    lot: 0,
+    unitNumber: "",
+    productDescription: ""
+  };
+
+  //When loading this page, retrive the data for this particular apartment from backend
+  componentDidMount() {
+    LocalApi.get(`${process.env.REACT_APP_BASEURL}/images/`)

```

Comment:
LocalApi uses localhost:3000 already. ${process.env.REACT_APP_BASEURL} is not needed here.

```
LocalApi.get("/images")
...
```

Reply:
Thanks! Fixed!